### PR TITLE
FIX: mender-artifact cp no longer renames the artifact.

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None

Signed-off-by: Ole Petter <ole.orhagen@cfengine.com>

### CODE REVIEW

**Comment by oleorhagen** [comment]
@kacf looks spurious to me

**Comment by oleorhagen** [comment]
@kacf  Well, this PR is simply for the artifact-name. But sure.

**Comment by oleorhagen** [comment]
Update names no longer change: 
https://github.com/mendersoftware/mender-artifact/pull/134

**Comment by oleorhagen** [comment]
@kacf good to go?

**Comment by kacf** [comment]
@oleorhagen: I'm a little confused. Which PR does what? There are four PRs, #131, #132, #133 and #134, and they all seem very similar. Can you close the unnecessary ones, so I can review the correct one(s)?

File: cli/mender-artifact/copy_test.go [hunk 1]
```diff
@@ -292,6 +293,33 @@ func TestCopy(t *testing.T) {
...
+				}
+				ar := areader.NewReader(f)
+				_, err = read(ar, ver, readScripts)
+				require.Nil(t, err)
+				// Verify that the artifact-name has not changed.
+				assert.Equal(t, "mender-1.1", ar.GetArtifactName())
+				// Cleanup
+				os.Remove("foo.txt")
+			},
+		},
```
Comment (pasinskim) [inline cli/mender-artifact/copy_test.go]: Nice test!

File: cli/mender-artifact/partition.go [hunk 2]
```diff
@@ -228,7 +228,7 @@ func (a *artifactExtFile) Close() error {
 	os.Remove(a.tmpf.Name())
 	if a.repack {
 		return repackArtifact(a.name, a.path,
-			a.key, filepath.Base(a.name))
+			a.key, "")
```
Comment (pasinskim) [inline cli/mender-artifact/partition.go]: Is that it? Nice fix then!

**Comment by pasinskim** [comment]
@oleorhagen is this one superseded by https://github.com/mendersoftware/mender-artifact/pull/134?

File: cli/mender-artifact/partition.go (hunk 2 above)
Comment (oleorhagen) [reply cli/mender-artifact/partition.go]: That is it :)

**Comment by oleorhagen** [comment]
Well, the fix is in #134 , but not the test. I think I'll just squash #134 and leave it as it is. Agree?

**Comment by kacf** [comment]
Superseded by #137.

**Pull request closed without merging**